src/Temporalio/Worker/WorkflowWorker.cs

```diff
@@ -167,7 +167,7 @@ private async Task HandleActivationAsync(IPayloadCodec? codec, WorkflowActivatio
                             // next activation which will be a remove job.
                             deadlockedWorkflows[act.RunId] = workflowTask;
                             throw new InvalidOperationException(
-                                $"Workflow with ID {act.RunId} deadlocked after {deadlockTimeout}");
+                                $"[TMPRL1101] Workflow with ID {act.RunId} deadlocked after {deadlockTimeout}");
```

Comment:
Maybe re-use this format string?

Reply:
Meh, just a couple of places for the string literal, I don't think it's worth extracting